allensdk/brain_observatory/ecephys/ecephys_project_cache.py

```diff
@@ -72,15 +72,15 @@ class EcephysProjectCache(Cache):
 
     def __init__(
             self,
-            fetch_api: EcephysProjectApi = EcephysProjectWarehouseApi.default(), 
+            fetch_api: EcephysProjectApi = EcephysProjectWarehouseApi.default(),
```

Comment:
I know this line was here before, but doesn't this mean that each project cache's default fetch api is this single, technically mutable, instance?

Reply:
Yes, this sounds correct, I only modified this line to appease the linter (which hates trailing whitespace).

It looks like the `default()` is setting the warehouse endpoint: https://github.com/AllenInstitute/AllenSDK/blob/c0d5d40be5c48e089b7121b2dcb4557504aa2dd4/allensdk/brain_observatory/ecephys/ecephys_project_api/ecephys_project_warehouse_api.py#L310

Reply:
I am only suggesting that the default value in the parameter list be None, and that the object's construction occur inside the method.

Reply:
Updated!